client/components/community-translator/translated-success.jsx

```diff
@@ -0,0 +1,19 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const TranslatedSuccess = ( { translationUrl } ) => (
+	<div className="community-translator__success">
+		<p>Thanks for contributing!</p>
+		<p>
+			Your translation has been submitted. You can view it on{' '}
```

Comment:
Should we localize these two strings? Also, I'm not sure what is `"You can view it on {' '}"`. Could you help me understand it?

Reply:
`{' '}` is magic. :) Actually no idea what it's doing there. I think my IDE added it to force a space. Let's delete it :)